### ISSUE TITLE
bug(cdkTargetMenuAim): directive does not really work

### ISSUE THREAD

**Issue by Beilinson** [angular/components#28986]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

According to the docs, the `cdkTargetMenuAim` allows for a "Smart Aim" behavior which calculates the intended slope of the mouse pointer and does not switch to another submenu unless some timeout is reached or the mouse deviates from the slope. This behavior does not seem to work at all (tested on several versions: 16.1.2, 17.3.6).

### Reproduction

StackBlitz link: https://stackblitz.com/edit/qwy1he?file=src%2Fexample%2Fcdk-menu-menubar-example.html
Steps to reproduce:
1. Try moving in a diagonal line from the `edit` menu item to the `cut`  menu item in the submenu. 
2. Doing so will immediately open the `share` submenu instead, no matter how fast the action is done (unless its faster than the refresh rate of the monitor)

### Expected Behavior

That moving the mouse along a diagonal line only opens a different subMenu if the mouse took too long to travel.

### Actual Behavior

The different submenus disappear immediately.

From my limited debugging I was able to glean that the issue seems to stem from the PointerTracker, where the `_getItemPointerEntries()` stream doesn't seem to get output any value on entering children menu items, while the exit is called every time, meaning the `previousElement` and `activeElement` are set as undefined for the duration of the mouse interaction with the menu items and submenus.

### Environment

- Angular: 17.3.6
- CDK: 17.3.6
- Browser(s): Chrome, Firefox
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by pimenovoleg** [comment]
I also don't see any examples in the documentation :(

is there an opportunity to raise the priority of this task?

**Comment by chrstnbrn** [comment]
@Beilinson I just noticed that your StackBlitz example doesn't import the CdkTargetMenuAim directive. After adding it, the directive works as expected I think (although you have to be quite fast).

```
@Component({
  selector: 'cdk-menu-menubar-example',
  exportAs: 'cdkMenuMenubarExample',
  styleUrl: 'cdk-menu-menubar-example.css',
  templateUrl: 'cdk-menu-menubar-example.html',
  standalone: true,
  imports: [
    CdkMenuBar,
    CdkMenuItem,
    CdkMenuTrigger,
    CdkMenu,
    CdkMenuGroup,
    CdkMenuItemCheckbox,
    CdkMenuItemRadio,
    CdkTargetMenuAim, // <-- This was missing
  ],
})
export class CdkMenuMenubarExample {}
```

**Comment by Beilinson** [comment]
@chrstnbrn youre right, in my actual project I did however import it but probably experienced it "not working" because it was simply too fast. Any chance to allow users to configure the behavior from the outside?

We ended up making our own trigger button with a custom timeout. but i'd love to get rid of that code and use the CDK exclusively
